From d9e0ffcde8f36cae76dffa9706a88d68be4d1f12 Mon Sep 17 00:00:00 2001
From: Sergii Piatakov <sergii.piatakov@globallogic.com>
Date: Thu, 15 Nov 2018 15:26:35 +0200
Subject: [PATCH 2/2] formatting: use tabs instead of spaces

Signed-off-by: Sergii Piatakov <sergii.piatakov@globallogic.com>
Signed-off-by: xczcv <alnazarchuk47@gmail.com>
---
 calculator.cpp | 12 ++++++++----
 calculator.h   |  6 +++---
 2 files changed, 11 insertions(+), 7 deletions(-)

diff --git a/calculator.cpp b/calculator.cpp
index 34f8d05..71110fc 100644
--- a/calculator.cpp
+++ b/calculator.cpp
@@ -2,14 +2,18 @@
 
 int Calculator::Add (double a, double b)
 {
-<<<<<<< HEAD
+
+
     return a + b;
-=======
+
 	return a + b + 0.5;
->>>>>>> 4ad40a3 (fix truncation error)
+
+
+	return a + b;
+
 }
 
 int Calculator::Sub (double a, double b)
 {
-    return Add (a, -b);
+	return Add (a, -b);
 }
diff --git a/calculator.h b/calculator.h
index 3740907..d59d596 100644
--- a/calculator.h
+++ b/calculator.h
@@ -3,9 +3,9 @@
 
 class Calculator
 {
-    public:
-        int Add (double, double);
-        int Sub (double, double);
+	public:
+		int Add (double, double);
+		int Sub (double, double);
 };
 
 #endif//CALCULATOR_H
-- 
2.43.0

